NP/Unicode.hs

```diff
@@ -57,6 +57,48 @@ greekData = [(Just 'a', "alpha", "α")
             ,(Just 'O', "Omega", "Ω")
             ]
 
+-- | Elder Futhark runes
+futhark :: [(String, String)]
+futhark = [(name, unicode) | (_,name,unicode) <- futharkData] ++
+        [ ([leading,shorthand],unicode)
+        | (Just shorthand,_,unicode) <- futharkData
+        , leading                    <- ['\'', 'f'] ]
```

Comment:
So as you might have guessed I copy pasted this from the greek version. Of course I made a mistake and thereby retroactively justifying my distrust of copy paste.